calculator/main.py

```diff
@@ -58,18 +72,26 @@ def get_history(calc):
         ie:
         ('2016-05-20 12:00:00', 'add', (1, 2), 3),
     """
-    pass
+    return calc['history']
 
 
 def reset_history(calc):
     """
     Resets the calculator history back to an empty list.
     """
-    pass
+    calc['history'][:] = []
```

Comment:
could you explain why you used `[:]` please? `calc['history'] = []` is sufficient